### DISCUSSION TITLE
building docker container [Jun 21]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2500] [Discord Threads]
> 🙋‍♂️ **wisethrall**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Sorry, no. We build it via a GitHub action. But this could be done if you can figure setting local dev env. Happy to take a PR in our docs  
#  
> 🙋‍♂️ **fjywan**  
#### looking forward to this!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/943892165107527690)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
